src/net.h

```diff
@@ -49,6 +50,9 @@ static const int TIMEOUT_INTERVAL = 20 * 60;
 static const unsigned int MAX_INV_SZ = 50000;
 /** The maximum number of new addresses to accumulate before announcing. */
 static const unsigned int MAX_ADDR_TO_SEND = 1000;
+/** The maximum # of bytes to receive at once */
```

Comment:
Perhaps this should remark that we allocate a buffer of this size on the stack, so making it too high can result in stack overflows? I do not know what the default stack size is on each platform but I'm guessing we wouldn't want this to be much larger than, say, 500kb.